City-of-Helsinki/drupal-hdbt
# UHF-8748: Add selections wrapper common component

# [UHF-8748](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8748)


## What was done


* Add SelectionsWrapper React common component and use it in all react searches
* Cleaning styles and translations from news archive

## How to install
* You need to have the following instances
  * https://github.com/City-of-Helsinki/drupal-helfi-kymp
  * https://github.com/City-of-Helsinki/drupal-helfi-rekry
  * https://github.com/City-of-Helsinki/drupal-helfi-tyo-yrittaminen
  * https://github.com/City-of-Helsinki/drupal-helfi-etusivu
  * https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus
* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-8748-add-selections-wrapper-common-component`
* Run `make drush-cr`

## How to test


* [ ] Check that filter bullets (selected filter values) still work correctly in all the searches:
  * [x] District and Project search
https://helfi-kymp.docker.so/fi/kaupunkiymparisto-ja-liikenne/kaupunkisuunnittelu-ja-rakentaminen/hae-suunnitelmia-ja-hankkeita
  * [x] Job search
https://helfi-rekry.docker.so/fi/avoimet-tyopaikat/etsi-avoimia-tyopaikkoja
  * [x] Linkedevents (this can also be tested in any instance by adding events paragraph to a landing page)
https://helfi-elo.docker.so/fi/yritykset-ja-tyo/tapahtumat
  * [x] News Archive
https://helfi-etusivu.docker.so/fi/uutiset
Check also that translations and styles work correctly
  * [x] School search
https://helfi-kasko.docker.so/fi/kasvatus-ja-koulutus
You have to create a Landing Page with School Search paragraph


[UHF-8748]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8748?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
